apis/operator/v1alpha1/telemetry_types.go

```diff
@@ -69,6 +69,7 @@ type TraceGatewaySpec struct {
 // LogSpec defines the behavior of the log gateway
```

Comment:
The comment has to be rephrased - it's not limited to the log gateway anymore.

Reply:
Documents updated